Assets/MixedRealityToolkit/_Core/Utilities/InputMappingAxisUtility.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    /// <summary>
+    /// Utility class for Unity's Input Manager Mappings.
+    /// Input from all types should be defined here for use 
+    /// though out the entire toolkit.
+    /// </summary>
+    public static class InputMappingAxisUtility
+    {
+        #region Nested Types
+
+        /// <summary>
+        /// Used to map AxisType from a useful name to the int value the InputManager wants.
+        /// </summary>
+        public enum MappingAxisType
+        {
+            KeyOrMouseButton = 0,
+            MouseMovement,
+            JoystickAxis
+        };
+
+        /// <summary>
+        /// Used to define an entire InputManagerAxis, with each variable defined by the same term the Inspector shows.
+        /// </summary>
+        public class InputManagerAxis
+        {
+            public string Name = "";
```

Comment:
`string.Empty` please

Reply:
Dang, that'll be from the old code.  Good catch